python-frontend/typeshed_serializer/build-with-docker.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+export USER_ID="$(id -u)"
+export GROUP_ID="$(id -g)"
```

Comment:
Unused code here

Reply:
it is used in docker-compose.yml user section